shared/types/CoveyTownSocket.d.ts

```diff
@@ -109,6 +109,12 @@ export interface TicTacToeMove {
   col: TicTacToeGridPosition;
 }
 
```

Comment:
comment here for why we are re-using a tictactoe game piece for future reference

Reply:
Resolved.